core/src/main/resources/hudson/logging/LogRecorderManager/sidepanel_pl.properties

```diff
@@ -1,8 +1,27 @@
-# This file is under the MIT License by authors
-
+# The MIT License
+#
+# Copyright (c) 2013-2017, Kohsuke Kawaguchi, Damian Szczepanik
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
 All\ Logs=Wszystkie Logi
 Back\ to\ Dashboard=Powr\u00F3t do tablicy
 Log\ Levels=Poziom logowania
 Logger\ List=Lista loger\u00F3w
 Manage\ Jenkins=Zarz\u0105dzaj Jenkinsem
-New\ Log\ Recorder=Nowe Nagrywanie Log\u00F3w
+New\ Log\ Recorder=Dodaj logera
```

Comment:
Notably, in English, we use *log recorder* as an aggregator of the output of one or more *loggers* (which are basically just a package/class name and logging level). Unfortunately this is somewhat inconsistent, as *Back to Loggers* (which shows the *system log* main page) and *Logger List* (which actually refer to *log recorders*) indicate.

Did you take that into account in your translation?